src/ol/format/jsonfeatureformat.js

```diff
@@ -26,7 +26,7 @@ ol.inherits(ol.format.JSONFeature, ol.format.Feature);
  * @return {Object} Object.
  */
 ol.format.JSONFeature.prototype.getObject_ = function(source) {
-  if (goog.isObject(source)) {
+  if (source !== null && typeof source === 'object') {
```

Comment:
It's safer here to just check `if (source !== null)`, because if it's a Node or Document, `typeof source` would probably be `'function'` and not `'object'`.